### DISCUSSION TITLE
Sovol SV06+ - does not use the default mesh ?

### DISCUSSION THREAD

**Discussion by Manfredo1952** [bassamanator/Sovol-SV06-firmware#87] [Q&A]
Just realise that (or it seems so at least) that the printer is not using the info/data of the default mesh. When I print a layer of say 0.24 mm, there is one area where the filament gets cut open... and I don't see the zsteppers movisng whilst printing ? What can I do to test/verify ?

regards

**Comment by bassamanator** [comment]
If kamp is disabled, the default mesh [is loaded](https://github.com/bassamanator/Sovol-SV06-firmware/blob/0ed21d3008d59383cc3287776dc35e7c76c85190/cfgs/misc-macros.cfg#L166C1-L166C1).

If you did not have a mesh named `default`, `print_start` would crash.

Therefore, if you're not crashing, and kamp is not enabled based on your observations (ignore the code), the default mesh is in fact being loaded.

I sometimes see marks on the top surfaces of my prints where the nozzle didn't quite lift high enough when travelling to another position, I think that's what you're experiencing. How bad are the cuts? Can you print issue free or is this affecting your ability to get a clean print?

**Comment by Manfredo1952** [comment]
Thanks for the comments.  I have done  a few more tests. After 2-3 reboots I can now see the zsteppers working and I have slowed down printing the first layer to 50 mm/s as well have increased flow rate for the first layer to 1,1 (1 otherwise), Looks a lot better now-  Thought I could print first layer with 125 as well,  may be not a good idea..  Kamps works very well. Thanks !
